ensembl/conf/json/macaca_fascicularis_vcf.json

```
@@ -8,10 +8,9 @@
       "assembly": "Macaca_fascicularis_6.0",
       "type": "remote",
       "use_as_source" : 1,
-      "filename_template" : "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/macaca_fascicularis/GCA_011100615.1/GCA_011100615.1_current_ids.vcf.gz",
-      "chromosomes": [
-        "1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","X"
-      ]
+      "source_version" : 4,
+      "filename_template" : "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_4/by_species/macaca_fascicularis/GCA_011100615.1/GCA_011100615.1_current_ids.vcf.gz",
+      "use_seq_region_synonyms": 1
```

HI @olaaustine , it seems the json we tested have this value as 0. Can you revert it back -
```suggestion
      "use_seq_region_synonyms": 0
```